canvas_sdk/effects/__init__.py

```diff
@@ -1 +0,0 @@
-from generated.messages.effects_pb2 import Effect, EffectType
```

Comment:
Does this break the imports elsewhere `from canvas_sdk.effects import Effect`

Reply:
i imagine it would.. honestly don't know how this change snuck in there. i'm going to revert it.

Reply:
also the banner alert effects are supposed to be in imported into this file too... something funky happened for sure! good catch

Reply:
mystery solved, it's a pre-commit modification. its also messing with the effects_pb2 files.

Reply:
Yeah, I actually just came across this and fixed it here: https://github.com/canvas-medical/canvas-plugins/pull/62

I hope you don't mind I'm going to merge that first.

Reply:
Reason for merging urgently: that PR fixes something that prevented us from releasing a new version of the pypi package.

Reply:
thanks!! i'll rebase here and fix conflicts